workers/vercel-edge-functions/README.md

```diff
@@ -0,0 +1,52 @@
+# JSON Response Example
```

Comment:
This file should probably be replaced with our usual minimal README for test projects.

Reply:
Totally forgot, thanks!